src/main/java/com/spotify/docker/client/messages/NetworkSettings.java

```diff
@@ -215,6 +228,17 @@ public Builder macAddress(final String macAddress) {
       return this;
     }
 
+    public Builder networks(final Map<String, AttachedNetwork> networks) {
+      if (networks != null) {
+        final ImmutableMap.Builder<String, AttachedNetwork> builder = ImmutableMap.builder();
```

Comment:
I think you can replace everything in this `if` with `this.networks = ImmutableMap.copyOf(networks);`